build.gradle

```diff
@@ -331,7 +338,7 @@ configurations.all {
     exclude group: 'javax.media', module: 'jai_imageio'
 }
 
-//compileScala.dependsOn(scalafmtAll)
+compileScala.dependsOn(scalafmtAll)
```

Comment:
This should be reviewed in the standup as I vaguely recall the reason for this was that it messed with compile times - so we added the jenkins check to catch it before it was fully commited

Reply:
agree. it basically change a lot of untouched files